src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -27,11 +75,11 @@ public void stop() {
         shooter.set(motorStop);
     }
 
-    public void turretUp() {
-        shooterServo.set(turretPower);
-    }
+    // public void turretUp() {
+    // shooterServo.set(turretPower);
+    // }
 
-    public void turretDown() {
-        shooterServo.set(-turretPower);
-    }
+    // public void turretDown() {
+    // shooterServo.set(-turretPower);
+    // }
```

Comment:
whats going on here? IS this the turret or hood?

Reply:
big misunderstanding, but this would be for the hood. alex is making seperate subsystem for hood.